src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -743,3 +743,48 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-navigation]
+== Navigation
+
+The *navigation page block* allows you to show navigation, which can easily navigate through different pages.
+
+image:compose-configuration/page-block-reference/config-navigation.png[...,49%,role="preview data-zoomable"]
+image:compose-configuration/page-block-reference/added-navigation.png[...,49%,role="preview data-zoomable"]
+
+.Additional configurations:
+[cols="2s,5a"]
+|===
+| [#page-block-navigation-appearance]#<<page-block-navigation-appearance,Appearance>>#
+|
+Defines the style of the navigation
+
+.Supported appearances:
+* Tabs
+* Pill
+* Small
+
+| [#page-block-navigation-justify]#<<page-block-navigation-justify,Justify>>#
+|
+Sets equal width for all elements on the navigation
+
+| [#page-block-navigation-alignment]#<<page-block-navigation-alignment,Alignment>>#
+|
+Defines the text alignment of each navigation item
+
+.Supported alignments:
+* Left
+* Center
+* Right
+|===
+
+[TIP]
+====
+You can define multiple navigation items for the navigation page blocks, which allows you to customize and display all supported navigation item type.
+====
```

Comment:
I'd consider moving this tip to line `751`.
I generally prefer to keep admonitions closer to the _main content_ because I find they're easily missed when placed under screenshots (you just scroll past them) or config options/tables (you again scroll past them).